driver-core/src/main/java/com/datastax/driver/core/NettySSLOptions.java

```diff
@@ -37,7 +39,13 @@ public NettySSLOptions(SslContext context) {
     }
 
     @Override
+    public SslHandler newSSLHandler(SocketChannel channel, InetSocketAddress address) {
+        return context.newHandler(channel.alloc(), address.getHostName(), address.getPort());
+    }
+
+    @Override
     public SslHandler newSSLHandler(SocketChannel channel) {
-        return context.newHandler(channel.alloc());
```

Comment:
Same here, the current implementation should be kept as is to avoid breaking user code.